conans/client/graph/install_graph.py

```diff
@@ -485,14 +485,9 @@ def get_errors(self):
             errors.append("There are some error(s) in the graph:")
         if invalid:
             for package in invalid:
-                node = package.nodes[0]
-                if node.cant_build and node.should_build:
-                    binary, reason = "Cannot build for this configuration", node.cant_build
-                else:
-                    binary, reason = "Invalid", node.conanfile.info.invalid
-                errors.append(f"{tab}- {node.ref}: {binary}: {reason}")
+                errors.append(f"{tab}- {package.pref}: Invalid configuration")
```

Comment:
I was thinking to simplify the message when there was no ``.nodes`` information (``conan graph build-order-merge``), but leave it as-is in the ``conan graph build-order`` command. But now I see the code I am fine with this simplification, a bit of loss of UI, but simpler code, good to me.